packages/react-instantsearch/rollup.config.js

```diff
@@ -46,7 +46,7 @@ const plugins = [
 
 const createConfiguration = ({ name, minify = false } = {}) => ({
   input: 'src/index.ts',
-  external: ['react'],
+  external: ['react', 'ai'],
```

Comment:
can be done later, but I think we should disable the whole chat component in UMD instead of just making the ai lib external.